src/ome.ts

```diff
@@ -26,8 +26,8 @@ export async function loadWell(config: ImageLayerConfig, grp: ZarrGroup, wellAtt
 
   if (acqIds.length > 1) {
     // Need to get acquisitions metadata from parent Plate
-    const platePath = grp.path.replace(`${row}/${col}`, '');
-    const plate = await openGroup(grp.store, platePath);
+    const plateUrl = grp.store.url.replace(`${row}/${col}`, '');
+    const plate = await openGroup(new HTTPStore(plateUrl));
```

Comment:
@will-moore perhaps we can add a special flag to check for the proxy.

```javascript
    const plateUrl = grp.store.url.replace(`${row}/${col}`, '');
    const store = new HTTPStore(plateUrl);
    const plate = await openGroup(grp.store.__nested ? nested(store) : store);
```

Reply:
Ah, I guess we don't don't need this here because we create the parent just to look at the attrs. 

Doesn't matter if it is nested or not since we don't use `plate` except to get the metadata.